frontend/src/components/AddUpdate/AddActivityUpdate.tsx

```diff
@@ -0,0 +1,81 @@
+                                                                        value={amountOfSkiingDays}
+                                                                        onChange={event => setAmountOfSkiingDays(Number(event.target.value))}/>
+            </label>
+            <label className="label-input-update"> Beauty Days:<input className="number-field"
+                                                                       type="number"
+                                                                       placeholder="Add additional Spa and Wellness days" min="0" step="1"
+                                                                       value={amountOfBeautyDays}
+                                                                       onChange={event => setAmountOfBeautyDays(Number(event.target.value))}/>
+            </label>
+            <label className="label-input-update"> Custom Activity Item: <input className="text-field"
+                                                                                 type="customActivityItem"
+                                                                                 placeholder="Add a additional Custom Activity Item"
+                                                                                 value={customActivityItem}
+                                                                                 onChange={event => setCustomActivityItem(event.target.value)}/>
+            </label>
+            <label className="label-input-update"> Amount of Custom Activity Item: <input className="number-field"
+                                                                                           type="number"
+                                                                                           placeholder="Add Amount of bought Custom Activity Items" min="0" step="1"
+                                                                                           value={amountOfCustomActivityItem}
+                                                                                           onChange={event => setAmountOfCustomActivityItem(Number(event.target.value))}/>
+            </label>
+            <label className="label-input-update"> Emissions of Custom Activity Item: <input className="number-field"
+                                                                                              type="number"
+                                                                                              placeholder="Add Emissions of your Custom Activity Item" min="0" step="0.1"
+                                                                                              value={customActivityItemEmission}
+                                                                                              onChange={event => setCustomActivityItemEmission(Number(event.target.value))}/>
```

Comment:
There is so much nearly the same. Can this be more abstract? Like a function Co2InputField("This is the name of the Item", "This should be the placeholder", onChangeFunction).

Reply:
More abstraction is on my to-do list. But the first goal is to achieve an initial running application. So some parts are just quick and dirty!